shapes/profile/app.shex

```diff
@@ -0,0 +1,36 @@
+PREFIX oidc: <http://www.w3.org/ns/solid/oidc#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schem: <http://schema.org/>
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX cert:  <http://www.w3.org/ns/auth/cert#>
+PREFIX ldp: <http://www.w3.org/ns/ldp#>
+PREFIX sp: <http://www.w3.org/ns/pim/space#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+
+<#id> {
+  a schem:SoftwareApplication
+    // rdfs:comment  "A software application. (from Schema.org)" ;
```

Comment:
I'd recommend we always provide explicit language tags on all comments, labels, messages, etc.
```suggestion
    // rdfs:comment  "A software application. (from Schema.org)"@en ;
```